detect_secrets/core/audit.py

```diff
@@ -542,6 +548,19 @@ def _print_context(  # pragma: no cover
 
     if error_obj:
         raise error_obj
+    elif not exclude_remediation_note:
+        print(
+            '{}'.format(
+                colorize(
+                    (
+                        'A potential secret was detected in this code.'
+                        ' If so, it should be remediated.'
```

Comment:
```suggestion
                        ' If so, you should select "Yes" below to mark it as valid secret and remediate it. The selection will be removed from baseline file by pre-commit hook once the secret is been removed from the file.'
```

Make it more wordy to explain even select "Yes", once removed / updated the raw secret in source file, the selection in baseline file will get removed.

Reply:
Thanks for the suggestion, the additional details definitely make things clearer.

I made some wording changes, let me know if this look good to you:
<img width="1409" alt="Screen Shot 2022-01-04 at 11 20 29 AM" src="https://user-images.githubusercontent.com/12804086/148090185-305e698c-51c7-4d44-a793-fc3e59c5736d.png">

Reply:
That looks great to me! @victoria-miltcheva